.github/workflows/Deploy.yml

```
@@ -1,6 +1,6 @@
 name: Deploy
 
-on:
+on: push
   push:
```

actionlint (1.7.3)

could not parse as YAML: yaml: line 4: mapping values are not allowed in this context

(syntax-check)